crates/starknet-server/src/api/http/endpoints/mint_token.rs

```diff
@@ -12,36 +14,44 @@ pub(crate) async fn get_fee_token() -> HttpApiResult<Json<FeeToken>> {
     Err(HttpApiError::GeneralError)
 }
 
+/// get the balance of the `address`
+fn get_balance(
+    starknet: &Starknet,
+    address: ContractAddress,
+) -> Result<BigUint, starknet_core::error::Error> {
+    let erc20_address = Felt::from_prefixed_hex_str(ERC20_CONTRACT_ADDRESS).unwrap();
+    let balance_selector =
+        starknet_rs_core::utils::get_selector_from_name("balanceOf").unwrap().into();
+    let new_balance_raw = starknet.call(
+        BlockId::Tag(BlockTag::Pending),
+        erc20_address,
+        balance_selector,
+        vec![Felt::from(address)], // calldata = the query balance
+    )?;
+
+    // format balance for output - initially it is a 2-member vector (low, high)
+    assert_eq!(new_balance_raw.len(), 2);
```

Comment:
Why not returning some error instead of asserting.
Is there a possible situation where the return data of call method is vector of 1 element

Reply:
This is more like a sanity check during development. If this fails, it means there is a bug and the code is not ready to be committed/released. Users won't ever see this (as long as we are properly testing the code).

Reply:
It's not a constant, but a result returned from external library. How to properly test it, so its certain that the user doesn't see it

Reply:
Ended up replacing it in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/99/commits/01266b4e7c0f5c017be15f792aa313121a3c9d7b